perllib/FixMyStreet/Script/Alerts.pm

```diff
@@ -126,10 +126,14 @@ sub send_alert_type {
             # this might throw up the odd false positive but only in cases where the
             # state has changed and there was already update text
             if ($row->{item_problem_state} && $last_problem_state ne $row->{item_problem_state}) {
-                my $cobrand_name = $report->cobrand_name_for_state($cobrand);
-                my $state = FixMyStreet::DB->resultset("State")->display($row->{item_problem_state}, 1, $cobrand_name);
+                my $update = '';
```

Comment:
This feels a bit janky to me, I guess it always works OK if Bexley have a block of text for the update. Otherwise, no email is sent.